src/app/layout.tsx

```diff
@@ -0,0 +1,70 @@
+"use client";
```

Comment:
The overall page structure should be done on the server side and the parts that need to be client components should be broken out into their own files that can be client side. 

The more we keep as server components, the faster it will be.